ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -180,14 +180,14 @@ private final class SqlLedger(
     checkpointQueue.complete()
   }
 
-  private def storeLedgerEntry(offsets: Offsets, entry: PersistenceEntry): Future[Unit] =
+  private def storeLedgerEntry(offset: Offset, entry: PersistenceEntry): Future[Unit] =
     ledgerDao
-      .storeLedgerEntry(offsets.offset, offsets.nextOffset, None, entry)
+      .storeLedgerEntry(offset, entry)
       .map(_ => ())(DEC)
       .recover {
         case t =>
           //recovering from the failure so the persistence stream doesn't die
-          logger.error(s"Failed to persist entry with offsets: $offsets", t)
+          logger.error(s"Failed to persist entry with offset: $offset", t)
```

Comment:
```suggestion
          logger.error(s"Failed to persist entry with offset: $offset.toApiString", t)
```

Reply:
Watch out for the missing braces before committing the suggestion. :wink: